docs/plan_autogen.md

```diff
@@ -0,0 +1,39 @@
+# Auto generated test plans
+
+The [test plans](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) in [src/webgpu/shader/](https://github.com/gpuweb/cts/tree/main/src/webgpu/shader), with a `unique-id`, are auto-generated by this [tool]( https://dawn.googlesource.com/tint/+/refs/heads/main/tools/src/cmd/get-test-plan/main.go). A typical auto-generated WGSL test plan:
+
+```
+g.test(<name>)
+  .uniqueId(<unique-id>)
+  .desc(
+    <url>
+    <description>
+  )
+  .params(u => u.combine('placeHolder1', ['placeHolder2', 'placeHolder3']))
+  .unimplemented();
+```
+
+- `name`: An arbitrary name.
+- `url`: The URL of the section this rule is extracted from, ie. `https://www.w3.org/TR/<version>/#<section-name>`. `section-name` is the section of the WGSL spec this rule is generated from.
+- `description`: The portion of WGSL spec text this rule is generated from.
+- `unique-id`: A token used for tracking and updating the auto generated test plans. `unique-id = sha1(<section-name> + <description>)`.
+
+# How to contribute
+
+To implement an auto-generated test plan:
+1. Choose an auto-generated test plan that is not currently under development, by looking up the `unique-id` in [this list](tbd).
+2. Create an issue using this [template](tbd). Make sure to include the `unique-id`.
+3. Implement the test plan. You may change `name` and `description`, however make sure to keep the `unique-id` as is.
```

Comment:
`this list`, ie. the list of open issues in `testing` component. Will be created in tint monorail.